### ISSUE TITLE
Invisible links are rendered visible in flowchart using elk renderer

### ISSUE THREAD

**Issue by tymokvo** [mermaid-js/mermaid#5244]
### Description

When using the `defaultRenderer: elk` configuration for the `flowchart`/`graph` diagram types, invisible links are rendered as normal links.

### Steps to reproduce

1. Use the config `%%{init: {"flowchart": {"defaultRenderer": "elk"}} }%%`
2. Create a graph with a `~~~` link between two nodes
3. Observe that the link is visible

### Screenshots

These were taken with the `develop` editor at `6939cf5`.

<img width="400" src="https://github.com/mermaid-js/mermaid/assets/12144028/c2cad291-fc81-41e0-b37e-4d04899652c1"/>
<img width="400" src="https://github.com/mermaid-js/mermaid/assets/12144028/9a3b8114-9897-4f84-8f2f-2c1f2fd7c2f2"/>

### Code Sample

```text
%%{init: {"flowchart": {"defaultRenderer": "elk"}} }%%
graph TD
    A --> B & C --> G
    I ~~~ G ~~~ H
```

### Setup

- Mermaid version: `v10.7.*`
- Browser and Version: [Chrome, Firefox]
  - Probably all, but I've only used these 2 

### Additional Context

Development editor example [here](https://develop.git.mermaid.live/edit#pako:eNo1jU9LxDAUxL9KeFBP3dIk_ZPkIKgLq1f1JLmEJt0W27TEFF1L-tnNdvEdhjfDj5kVmkkbEJAka297L9AqoR2m76ZTzkvYvTatWgb_aqw2zrhrKsEMnxJCQCFJpD07NXfo_SgtiveADod79Iju0NP-nW7xC9q2DZ12fYYURuNG1es4vl4BCb4zo7m1a-VivbQhcmrx09vFNiC8W0wKy6yVN8dexdURRKuGr5jOyn5M0_gPRQtihR8QhGa4oAzXPMcVxpSwFC4gyoyVhGNSkJLzgtU0pPC7F-RZTSvKCs4wK-uK5HX4A9E7VzI).

**Comment by TWiStErRob** [comment]
Why does [this](https://github.com/mermaid-js/mermaid/pull/4113) not work in elk?

**Comment by TWiStErRob** [comment]
Actually, @tymokvo `v11.0.0-alpha.7+7bcba29 Live Editor` seems to work, although the elk style is gone?

**Comment by tymokvo** [comment]
Agreed, the latest develop editor does not seem to be respecting the `elk` renderer.
